app/routes/application.js

```diff
@@ -0,0 +1,3 @@
+import ApplicationRouteMixin from 'simple-auth/mixins/application-route-mixin';
+
+export default Ember.Route.extend(ApplicationRouteMixin);
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'Ember' is not defined.